CyberHealth/CyberHealth/settings.py

```diff
@@ -212,6 +217,237 @@
     r"^https?:\/\/cyberhealth\.containers\.piwik\.pro$"
 ]
 
+TINYMCE_JS_URL = os.path.join(STATIC_URL, "tinymce/tinymce.min.js")
+TINYMCE_JS_ROOT = os.path.join(STATIC_ROOT, "tinymce/tinymce.min.js")
+
+TINYMCE_DEFAULT_CONFIG = {
```

Comment:
this is clearly madness but as discussed w Joel it seems like this is a necessary evil